<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushManager</class>
 <widget class="QFrame" name="BrushManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>55</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>5</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Brush Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="brushSizeInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="overwriteCheckbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Paint over existing tiles</string>
     </property>
     <property name="text">
      <string>Overwrite Tiles</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="smartBrushGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Smart Brush</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="smartBrushCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addSmartBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/addvalue.png</normaloff>:/Icons/addvalue.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editSmartBrushes">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/pencil.png</normaloff>:/Icons/pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteSmartBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/removevalue.png</normaloff>:/Icons/removevalue.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="scatterBrushGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Scatter Brush</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="scatterBrushCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addScatterBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/addvalue.png</normaloff>:/Icons/addvalue.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editScatterBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/pencil.png</normaloff>:/Icons/pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteScatterBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/removevalue.png</normaloff>:/Icons/removevalue.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="replacerGroup">
     <property name="title">
      <string>Replacer Brush</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="replacerBrushCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addReplacerBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/addvalue.png</normaloff>:/Icons/addvalue.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editReplacerBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/pencil.png</normaloff>:/Icons/pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteReplacerBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/removevalue.png</normaloff>:/Icons/removevalue.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="matrixBrushGroup">
     <property name="title">
      <string>Matrix Brush</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="matrixBrushCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addMatrixBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/addvalue.png</normaloff>:/Icons/addvalue.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editMatrixBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/pencil.png</normaloff>:/Icons/pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteMatrixBrush">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../Icons.qrc">
          <normaloff>:/Icons/removevalue.png</normaloff>:/Icons/removevalue.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
